### ISSUE TITLE
Research how open source foundations are defining community participation metrics

### ISSUE THREAD

**Issue by missaugustina** [BonnyCI/mateys-ahoy#1]
TL;DR: How do others currently define individual participation metrics?

This is part of a larger effort to understand how individuals are contributing to open source projects. Ultimately we need to quantify this activity. Open source projects are often run by non-profit open source foundations that need to justify their existence in order to procure funding. One thing many open source foundations care about is tracking contributor activity. This might include looking at the rate of people joining the community (new contributors) and the rate of people leaving the community, as well as how long someone has been involved in the community. They might also provide some way of ranking or rating the contributors. The goal of this issue is to discover how open source communities are currently quantifying contributor activity. 

**For each article you look at**
- Create a wiki page to capture research summaries/ideas/etc for this ticket (https://github.com/BonnyCI/mateys-ahoy/wiki -> New Page)
- For each article you read, put a bullet point/link in the wiki
- some kind of summary of what you thought the main point (relevant to us) (if it's worth it)
- ideas you maybe got
- ways you might improve on what they suggested (if you have any)
- questions - "what are they even talking about?" "why is this important to them?" "what question are they really trying to answer here?" with links to any relevant information that inspired

**Some suggestions:**

- https://web.archive.org/web/20130529020832/http://wiki.meego.com/Metrics/Dashboard
- https://github.com/mozilla/participation-metrics-org
- https://wiki.mozilla.org/ParticipationSystems/Participation_Metrics
- https://opensource.com/business/15/12/top-5-open-source-community-metrics-track
- https://osleadershipsummit2017.sched.com/event/9LeD/community-dashboards-for-linux-foundation-projects-from-open-to-inner-source-metrics-jesus-gonzalez-barahona-daniel-izquierdo-bitergia
- http://www.sciencedirect.com/science/article/pii/S0164121213002744
- https://events.linuxfoundation.org/images/stories/pdf/lceu2012_foster.pdf
- http://blog.symphony.foundation/2016-by-the-numbers
- https://gist.github.com/rbowen/7483b77276a5dbe3311f
- https://bitergia.com/

**Comment by missaugustina** [comment]
This will be assigned to @SarahZuk as soon as she has org access. I will also be helping out with this!

**Comment by missaugustina** [comment]
This is how Github counts contributions: https://help.github.com/articles/why-are-my-contributions-not-showing-up-on-my-profile/

**Comment by missaugustina** [comment]
@SarahZuk is updating the following wiki page: https://github.com/BonnyCI/mateys-ahoy/wiki/Participation-Metrics-Research

**Comment by missaugustina** [comment]
@SarahZuk one comment on the wiki page, thanks for starting that! Generally don't worry so much about the technical details of the articles. What I want you to focus on specifically is *what* they are measuring and more at a high level how they are measuring it. For instance, if an article indicates they measure the number of contributors, then describe how they count the number of contributors. If there is insufficient information in the article, then indicate that (eg, they say they count the number of contributors but I couldn't find more information on how they get that).
